src/interfaces/ast/utils.ts

```diff
@@ -1,8 +1,11 @@
 import { splitLimit } from './split';
 
 export const IDENTIFIER_RE = /^[_a-zA-Z][_a-zA-Z0-9]*$/;
+export const IDENTIFIER_RE_SOURCE = IDENTIFIER_RE.source;
 export const DOCUMENT_NAME_RE = /^[a-z][a-z0-9_-]*$/;
+export const DOCUMENT_NAME_RE_SOURCE = DOCUMENT_NAME_RE.source;
 export const PROVIDER_NAME_RE = /^[a-z][_\-0-9a-z]*$/;
```

Comment:
Seems like `PROVIDER_NAME_RE` is duplicate of `DOCUMENT_NAME_RE`.

Reply:
indeed